app/controllers/landing_controller.rb

```diff
@@ -1,6 +1,7 @@
 class LandingController < ApplicationController
 
   def index
+    @projects = Project.all 
```

Comment:
@antciccone are we sure we want to render all projects on the landing page? Maybe we should have a select number of 'highlighted' projects that will become a part of the carousel? Maybe if we have time we could have an admin privilege of making a project 'special' in some sense and then the carousel could call on the projects of that scope or something?